app/assets/javascripts/fontawesome_config.js

```diff
@@ -41,6 +41,7 @@ import {
   faUserMinus,
   faUserPlus,
   faWarning,
+  faMinus,
```

Comment:
This should go in alphabetical order with the other icons (makes it easier to keep track). Same below. Good catch on the `fawWarning`!